src/Clash/Cores/Ethernet/Icmp/IcmpTypes.hs

```diff
@@ -1,17 +1,17 @@
 {-|
-Module      : Clash.Cores.IP.ICMPTypes.hs
+Module      : Clash.Cores.Ethernet.Icmp.ICMPTypes.hs
 Description : Provides various data types, aliases and constants for ICMP.
 -}
-module Clash.Cores.IP.IcmpTypes
```

Comment:
This is so small that it's better to just move this to the `Icmp` module itself.